ARCs/arc-0002.md

```diff
@@ -33,14 +39,24 @@ where:
          * be at least 5 characters long
          * be at most 32 characters long
     * Names starting with `a/` and `af/` are reserved for the Algorand protocol and the Algorand Foundation uses.
+
+- `<arc-number>` is the number of the ARC with 0 padding:
+  - Regexp to satisfy: `0\d{3}|\d{4}`
+  - Only contain a four-digit number, either zero-padded or not, matching:
+    - 0\d{3} for zero-padded numbers (e.g., 0001, 0456).
+    - \d{4} for non-padded four-digit numbers (e.g., 1234, 5678).
+    - Be exactly four digits long.
```

Comment:
I don't understand why the regex `0\d{3}|\d{4}`. Doesn't `0\d{3}` match a subset of what `\d{4}` matches? If so, couldn't the whole regex be simplified to `\d{4}`?